biome.json

```diff
@@ -4,8 +4,8 @@
 		"include": [
 			"optuna_dashboard/ts/**/*.ts",
 			"optuna_dashboard/ts/**/*.tsx",
-			"typescript_tests/**/*.ts",
-			"typescript_tests/**/*.tsx",
+			"optuna_dashboard/ts/tests/**/*.ts",
+			"optuna_dashboard/ts/tests/**/*.tsx",
```

Comment:
Maybe we can just delete these (I think these are already included into two lines above these).

Reply:
Ah, you are right 🙏